README.md

```diff
@@ -60,6 +60,38 @@ Run the TypeScript linter.
 gulp tslint
 ```
 
+## Releases
+
+Increment all versions within S3Commander to new version (Ex. 0.12.1). Currently
+these are stored under the name s3commander in:
+
+```
+s3commander/packagelock.json
```

Comment:
package-lock.json is an autogenerated file: https://docs.npmjs.com/files/package-lock.json. As such it shouldn't be manually updated.